### ISSUE TITLE
Unable to use controllers I created

### ISSUE THREAD

**Issue by vonvick** [ingeniasoftware/luthier-ci#25]
I am unable to use the controller that I have created .
I have this in my route/web.php file
```php
$route = Luthier\Route::getRoutes();
```
I also defined some route like so:
```php
Route::post('admin/login', 'somecontroller@somemethod');
Route::get('admin', 'somecontoller@anothermethod', ['namespace' => 'xxxx']);

```

When I visit `/admin` on my browser I get this:
![screen shot 2018-08-20 at 5 49 19 pm](https://user-images.githubusercontent.com/21035558/44354309-666e6180-a4a1-11e8-901b-67a4f6297a40.png)

Also, going to the default route `/` gives me this:
![screen shot 2018-08-20 at 5 50 10 pm](https://user-images.githubusercontent.com/21035558/44354347-8736b700-a4a1-11e8-8a00-54389517024b.png)

What am I doing wrong?

Also, from your documentation, you talked about `Initialization`:
Can you explain how to initialize the package? What do I need to run to initialize it?

**Comment by andersonsalas** [comment]
Howdy!

You are using the Luthier CI routing in a wrong way. In your `application/config/routes.php` the content must be this:

```php
<?php

defined('BASEPATH') OR exit('No direct script access allowed');

$route = Luthier\Route::getRoutes();
```

You can find more information about the usage of Luthier CI [here](https://luthier.ingenia.me/ci/en/).

Also, in order to use friendly URLs, you must enable the Apache `mod_rewrite`.

**Comment by vonvick** [comment]
@andersonsalas Thanks for reaching out. I made a mistake. I have 
```php
$route = Luthier\Route::getRoutes();
```
 in my `config/routes.php` file and not in the `routes/web.php` file. But it is not working. What I have in the `routes/web.php` file is this:
```php
Route::post('admin/login', 'somecontroller@somemethod');
Route::get('admin', 'somecontoller@anothermethod', ['namespace' => 'xxxx']);
```
I created the files and folders manually.
Also, how do I generate those folders and files? Do I have to create them manually or is there a command to generate them?
